.importjs.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  importDevDependencies: true,
+  environments: ['node', 'browser'],
+  declarationKeyword: 'import',
+};
```

Comment:
It's an editor plugin that @trotzig and I hack on. Check it out! https://github.com/galooshi/import-js